src/tests/ftest/harness/core_files.yaml

```diff
@@ -2,14 +2,11 @@ hosts:
   test_servers: 2
 timeout: 120
 server_config:
+  system_ram_reserved: 2
```

Comment:
This one is interesting as the same test yaml file is used for two different test cases where one is run on VMs and the other is run on HW.  The test just starts the servers and crashes one of the engines to create a core file.  I suspect that leaving this at 2 shouldn't be a problem, but I wanted to check with you.

Reply:
thanks for the notice